docs/usage/automerge-configuration.md

```diff
@@ -0,0 +1,146 @@
+
+```json
+{
+  "packageRules": [
+    {
+      "updateTypes": ["minor", "patch"],
+      "matchCurrentVersion": "!/^0/",
+      "automerge": true
+    }
+  ]
+}
+```
+
+The `matchCurrentVersion` setting above is a rule to exclude any dependencies which are pre-1.0.0 because those can make breaking changes at _any_ time according to the SemVer spec.
+
+## Automerging and scheduling
+
+Automerging is particularly beneficial if you have configured a schedule, because Renovate on its own may be able to automerge the majority of your updates.
+And this is especially so if your repository needs rebasing, e.g. because you use lock files.
+e.g. let's say you have dependencies `abc` and `xyz` with upgrades, and you use a `yarn.lock` file.
+
+- At the start of the schedule, `Renovate` will create branches for `abc` and `xyz` upgrades, including `yarn.lock` updates
+- After `abc` passes tests, `Renovate` will automerge it to `master`
+- The `xyz` branch probably now has `yarn.lock` conflicts
+- Renovate will immediately check all other branches and rebase them
+- The change to `xyz` branch will trigger another round of CI tests
+- After the updated `xyz` branch passes, Renovate will automerge it too
+
+This is a lot better than you waking up to two PRs and then having to deal with conflicts yourself after you merge the first one.
+
+## Branch vs PR automerging
+
+Even if you automerge PRs, you are likely to still get notification noise - one when the PR is created and another when it is merged.
+For this reason we recommend you consider setting `automergeType=branch` which will mean:
+
+- Renovate first creates a branch and no PR
+- If tests pass, Renovate pushes a commit directly to the base branch without PR
+- If tests fail, Renovate raises a PR for you to review
+
+The result is that passing updates are essentially "silent" - the only sign of them are the commits to your `master` branch.
+If you have enabled branch protection which prevents Renovate from automerging directly to the base branch, then this won't work and you should stick with the default PR-based automerging instead.
+
+## Assignees and Reviewers
+
+When automerge is enabled on a PR, Renovate will _not_ add assignees or reviewers at PR creation time, in order to decrease notifications noise a little.
+If tests subsequently _fail_, making automerge not possible, then Renovate will then add the configured assignees and/or reviewers.
+
+## Frequent problems and how to resolve them
+
+### Automerge not enabled correctly in config
+
+Sometimes, the reason Renovate is not automerging a PR is because of a configuration mistake.
+You can confirm that Renovate knows you want it to automerge by seeing if the PR body includes the text "Automerge: Enabled".
+If you see "Automerge: Disabled by config" then it means you need a config change for automerge to work.
```

Comment:
```suggestion
Sometimes, the reason Renovate is not automerging a PR is because of a configuration mistake.
You can confirm that Renovate knows you want it to automerge by checking if the PR body includes the text "Automerge: Enabled".
If you see "Automerge: Disabled by config" then it means you need a config change for automerge to work.
```